wolfcrypt/src/sha.c

```diff
@@ -36,16 +36,25 @@
 #ifdef HAVE_FIPS
 	int wc_InitSha(Sha* sha)
 	{
+        if (sha == NULL) {
+            return -173;
```

Comment:
This should return the error code enum instead of the specific error code (ie: BAD_FUNC_ARG instead of -173)